src/backend/marsha/settings.py

```diff
@@ -158,6 +158,9 @@ class Base(Configuration):
         "EXCEPTION_HANDLER": "marsha.core.views.exception_handler",
         "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
         "PAGE_SIZE": 50,
+        "DEFAULT_THROTTLE_RATES": {
+            "anon": "2/minute",
```

Comment:
`2` it's a little bit harsh @sampaccoud , you have most of the time 3 attempts to use a valid PIN, we are in a similar case IMO

Reply:
I'm ok with 3 of course :+1: For a known device that is.